.github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'yarn'
       - run: yarn install --frozen-lockfile
-      - run: yarn lint
+#      - run: yarn lint
```

Comment:
Is this intentionally left commented out?
Required -> restore
Not required -> delete